plugins/balloontoolbar/plugin.js

```diff
@@ -376,6 +376,21 @@
 	}
 
 	ContextManager.prototype = {
+
+		/**
+		 * Creates {@link CKEDITOR.plugins.balloontoolbar.context} from the given options.
```

Comment:
I don't think we should repeat returned type explicitly here. It's enough it's in the `@returns` tag.

Let's just make it something like "Creates a toolbar context", so that we reduce change that future maintainer will change return value in one place, and not the other.

It would be also nice to add the information that this method (as opposed to add method) creates, adds and returns the context.

I would also love to see a short code listing here (yea I know that one was in the `editor.balloonToolbars` property, but I think it won't do harm).